src/helpers/reports.js

```diff
@@ -11,114 +11,224 @@ import { getGrade } from '@bdelab/roar-utils';
 export const taskDisplayNames = {
   letter: {
     name: 'Letter',
+    publicName: 'ROAR - Letter',
+    studentFacingName: 'Letter',
     extendedTitle: 'ROAR - Letter',
     extendedName: 'Letter Names and Sounds',
     order: 1,
   },
   'letter-es': {
     name: 'Letra',
+    publicName: 'ROAR - Letra',
+    studentFacingName: 'Letra',
     extendedTitle: 'ROAR - Letra',
     extendedName: 'Letter Names and Sounds',
     order: 2,
   },
-  pa: { name: 'Phoneme', extendedTitle: 'ROAR - Phoneme', extendedName: 'Phonological Awareness', order: 3 },
+  pa: {
+    name: 'Phoneme',
+    publicName: 'ROAR - Phoneme',
+    studentFacingName: 'Phoneme',
+    extendedTitle: 'ROAR - Phoneme',
+    extendedName: 'Phonological Awareness',
+    order: 3,
+  },
   'pa-es': {
     name: 'Fonema',
+    publicName: 'ROAR - Fonema',
+    studentFacingName: 'Fonema',
     extendedTitle: 'ROAR - Fonema',
     extendedName: 'Phonological Awareness',
     order: 4,
   },
-  swr: { name: 'Word', extendedTitle: 'ROAR - Word', extendedName: 'Single Word Recognition', order: 5 },
-  'swr-es': { name: 'Palabra', extendedTitle: 'ROAR - Word', extendedName: 'Single Word Recognition', order: 6 },
-  sre: { name: 'Sentence', extendedTitle: 'ROAR - Sentence', extendedName: 'Sentence Reading Efficiency', order: 7 },
-  'sre-es': { name: 'Frase', extendedTitle: 'ROAR - Frase', extendedName: 'Sentence Reading Efficiency', order: 8 },
+  swr: {
+    name: 'Word',
+    publicName: 'ROAR - Word',
+    studentFacingName: 'Word',
+    extendedTitle: 'ROAR - Word',
+    extendedName: 'Single Word Recognition',
+    order: 5,
+  },
+  'swr-es': {
+    name: 'Palabra',
+    publicName: 'ROAR - Palabra',
+    studentFacingName: 'Palabra',
+    extendedTitle: 'ROAR - Word',
+    extendedName: 'Single Word Recognition',
+    order: 6,
+  },
+  sre: {
+    name: 'Sentence',
+    publicName: 'ROAR - Sentence',
+    studentFacingName: 'Sentence',
+    extendedTitle: 'ROAR - Sentence',
+    extendedName: 'Sentence Reading Efficiency',
+    order: 7,
```

Comment:
Is it possible to use your taskDictionary() to define some of these fields?

One idea might be to start storing all of these fields in the task document intself, and then write a function(s) that grabs the documents and sets these fields. 

This would make it easier to modify task documents in the future and not have to change so many hard-coded values.

Just an idea for the future, nothing we need to do immediately.

Reply:
These fields are not being used to display titles on the front-end – all public names are pulled from the Firestore entry. Those entries are a relic from before and I thought I would leave them in for reference.